src/main/java/ru/yandex/qatools/allure/cucumberjvm/AllureRunListener.java

```diff
@@ -49,45 +51,102 @@ public void testRunStarted(Description description) {
     }
 
     /**
+     * Find features level
+     *
+     * @param description {@link Description} Description to search where
+     * @return {@link List<Description>} features description list
+     * @throws IllegalAccessException
+     */
+    private List<Description> findFeaturesLevel(List<Description> description)
+            throws IllegalAccessException {
+        if (description.isEmpty()) {
+            return new ArrayList<>();
+        }
+        Object readField = FieldUtils.readField(description.get(0), "fUniqueId", true);
+        if (readField instanceof Feature) {
+            return description;
+        } else {
+            return findFeaturesLevel(description.get(0).getChildren());
+        }
+
+    }
+
+    /**
+     * Find test classes level
```

Comment:
Parent description is multilevel. And its levels depends on several cases. So I should search right level to interact with right objects in several contexts.